### PR TITLE
piwww: Verify proposals with pi types

### PR DESCRIPTION
**Pull request by thi4go**
Adds `verifyProposal` back to `piwww` commands with `pi` api types. Renames `proposalpaywall` to `userproposalpaywall`.

### CODE REVIEW

File: politeiawww/cmd/piwww/proposals.go [hunk 1]
```diff
@@ -78,12 +78,24 @@ func (cmd *proposalsCmd) Execute(args []string) error {
...
+	for _, p := range reply.Proposals {
+		err = verifyProposal(p, vr.PubKey)
+		if err != nil {
+			return fmt.Errorf("unable to verify proposal %v: %v",
+				p.CensorshipRecord.Token, err)
+		}
+	}
 
-	// TODO verify proposals
+	err = shared.PrintJSON(reply)
```
Comment (lukebp) [inline politeiawww/cmd/piwww/proposals.go]: Put this up before the proposal verification.

**Review by lukebp — CHANGES_REQUESTED** [review]

File: politeiawww/cmd/piwww/proposals.go (hunk 1 above)
Comment (thi4go) [reply politeiawww/cmd/piwww/proposals.go]: done

File: politeiawww/cmd/piwww/piwww.go [hunk 2]
```diff
@@ -90,8 +72,24 @@ type piwww struct {
 	VoteSummaries   voteSummariesCmd   `command:"votesummaries"`
 	VoteInventory   voteInventoryCmd   `command:"voteinventory"`
 
-	// TODO rename to reflect that its a users route
-	ProposalPaywall proposalPaywallCmd `command:"proposalpaywall"`
+	// User commands
+	UserNew                userNewCmd                   `command:"usernew"`
```
Comment (lukebp) [inline politeiawww/cmd/piwww/piwww.go]: Don't move all of these commands.  Just add the new command to the existing list.

**Review by lukebp — CHANGES_REQUESTED** [review]
Looks good other than the formatting comment.

File: politeiawww/cmd/piwww/piwww.go (hunk 2 above)
Comment (thi4go) [reply politeiawww/cmd/piwww/piwww.go]: ops another rebase mistake, mb. fixed this

**Review by lukebp — APPROVED** [review]

**Pull request merged**